src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -13,9 +13,11 @@ public class XmlSystemPathResolverTests
         private const int k_getUniqueFileNameAttempts = 10;
 
         [Fact]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]  //[ActiveIssue(13121)]  // Access to path is denied in UWP
         public static void TestResolveRelativePaths()
         {
+            if (PlatformDetection.IsWinRT)  // Access to path not authorized inside an AppContainer
```

Comment:
But this is expected and getting handled further down right? 
Or is it the File.Open that's throwing?